docs/development.md

```diff
@@ -0,0 +1,172 @@
+| MAILER_EMAIL                | String             | info@cmp.io                 | Email address for email sending                                                                                  |
+| MAILER_FROM                 | String             | Consent Management Platform | Name for email address for email sending                                                                         |
+| SENTRY_DSN                  | String             | https://hash@sentry.io/123  | -                                                                                                                |
+| GTM_CONTAINER_ID            | String             | -                           | Optional, GTM will be initialized only if the variable is passed.                                                |
+| API_DOCS_ENABLED            | Boolean            | 1                           | Optional, enables OpenApi schema and Swagger UI.                                                                 |
+
+All ENV variables are also listed in the [.env.dist](../.env.dist) file.
+
+## Frontend
+
+As far as the visual side is concerned, the application is based mainly on Tailwind CSS and Alpine.js components.
+The final build is created using Webpack encore and PostCSS.
+
+You can rebuild the assets with following command:
+
+```sh
+$ make install-assets
+```
+
+The command internally runs `yarn install` and webpack encore in the production mode.
+
+Alternatively, the build can be run manually directly inside the container:
+
+```sh
+$ docker exec -it cmp-app sh
+$ yarn run encore prod # or `dev` 
+```
+
+## Fixtures
+
+Default demo data is automatically imported when the application is [installed](../README.md#installation).
+If you want to restart the data, you can do id by running the following command:
+
+```sh
+$ make data
+```
+
+Alternatively:
+
+```sh
+$ docker exec -it cmp-app sh
+$ bin/console doctrine:fixtures:load
+```
+
+:exclamation: Do not run fixtures on production!
+
+## Sending emails
+
+By default, emails are sent using the PHP function `mail()`. However, the `sendmail` extension is not installed in the container and therefore emails are not sent.
+For this reason, you need to configure the SMPT server through the ENV variables.
+For example, Google's SMTP server configuration would look something like this:
+
+```sh
+SMTP_ENABLED=1
+SMTP_HOST=smtp.gmail.com
+SMTP_PORT=465
+SMTP_USERNAME=your email address
+SMTP_PASSWORD=your token
+SMTP_SECURE=ssl
+```
+
+If the debug mode is enabled (`APP_DEBUG=1` or the user has set the `APP_DEBUG_COOKIE_SECRET` cookie in the browser), the emails are ALWAYS saved to local files and can be viewed in the Tracy Bar
+
+<img src="images/tracy-bar-mails.png" alt="Emails in the Tracy Bar" width="400">
+
+## API and integration
+
+The application exposes out API to communicate with the cookie bar.
```

Comment:
```suggestion
The application exposes our API to communicate with the cookie widget.
```

We should use "widget" instead of "bar". Tracy bar is OK 🙂